packages/nimble-components/src/table/specs/editable-table-ixd.md

```diff
@@ -0,0 +1,129 @@
+
+- When the table is used for displaying static or read-only data.
+- When the user requires guidance entering or modifying data and would be best served by a form or other UX pattern.
+- When the application requires spreadsheet features like pasting or filling data into a range of cells
+
+## Requirements
+
+- Inline editing support for table cells
+- Validation and error handling for input data
+- Initiate cell editing via keyboard or mouse
+- Support for strings and numbers (1st priority)
+- Support for enumerated values (2nd priority)
+- All cells within a particular column share the same edit view and configuration
+- Compatibility with assistive technologies (e.g., screen readers)
+
+### Out of scope
+
+- Support for complex data types (e.g., dates, times, custom objects)
+- Advanced spreadsheet functionalities (e.g., formulas, cell references)
+- Real-time collaboration or multi-user editing
+- Custom cell styling or formatting
+
+## Anatomy
+
+![Editable table anatomy](./spec-images/editable-cell-anatomy.png)
+
+| Element               | Description                                                              |
+| --------------------- | ------------------------------------------------------------------------ |
+| Static cell value     | Non-editable display of cell value                                       |
+| Action button         | Button to open cell context menu. Appears on row hover or keyboard focus |
+| Editable cell control | Text or numeric control that supports entering or editing cell data      |
+
+## Behavior
+
+### States
+
+![Editable cell states for text](./spec-images/text-editable-cell-states.png)
+![Editable cell states for numeric](./spec-images/numeric-editable-cell-states.png)
+![Editable cell states for select](./spec-images/select-editable-cell-states.png)
+![Editable cell states for checkbox](./spec-images/checkbox-editable-cell-states.png)
+
+#### Error State
+
+![Editable cell error states](./spec-images/editable-cell-error-data.png)
+
+### ARIA Considerations
+
+- Maintain compatibility with existing keyboard navigation behavior
+- Consider resolving [existing ARIA gaps](https://github.com/ni/nimble/issues/2285)
+- Consider adding `aria-readonly` for non-editable cells
+
+### Mouse Interactions
+
+![Editable cell mouse navigation](./spec-images/editable-cell-mouse-navigation.png)
+
+- Editable table cells show the same row hover state as non-editable cells on mouse hover.
+    - I.e. When hovering over a row, the action menu button appears. Depending on the table's selection mode, the row may be highlighted.
+- The cursor will change from an *arrow cursor* when hovering over editable cells
+    - To a *text cursor* when hovering over cells with editable text or numeric controls
+    - To a *hand cursor* when hovering over select or boolean controls
+- Single clicking editable cell shows the *edit focus* state.
+- Clicking away from a focused editable cell sets the value and removes *edit focus*.
```

Comment:
To be crystal clear, when you click on a non-editable cell while an editable cell has focus, _and_ the row is selectable, do we expect the row to become selected? This is what I expect to happen with no extra work.